lend/src/components/Common/LendModal.tsx

```diff
@@ -39,15 +41,66 @@ interface InfoModalProps {
 }
 
 const LendModal: React.FC<InfoModalProps> = ({ show, onHide, content }) => {
+  const [rbnClaimStep, setRbnClaimStep] = useState<ClaimRbnPageEnum>(
+    ClaimRbnPageEnum.CLAIM_RBN
+  );
+  const renderRbnClaimTitle = useCallback((rbnClaimStep: ClaimRbnPageEnum) => {
+    switch (rbnClaimStep) {
+      case ClaimRbnPageEnum.CLAIM_RBN:
+        return "CLAIM RBN";
+      case ClaimRbnPageEnum.TRANSACTION_STEP:
+        return "CLAIMING RBN";
+      case ClaimRbnPageEnum.SUCCESS_STEP:
+        return "RBN CLAIMED";
+    }
+  }, []);
+
+  const renderContent = useCallback(
+    (content?: ModalContentEnum) => {
+      switch (content) {
+        case "ABOUT":
+        case "COMMUNITY":
+        case "CONNECT WALLET":
```

Comment:
Could you convert these to reference the Enum instead?

e.g.: `case ModalContentEnum.ABOUT`